app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] As shown in [erd diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), kindly use author like this `belongs_to :author, class_name: 'User'`